index.html

```diff
@@ -0,0 +1,447 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Amine Smahi - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <div class="logo">
+        <a href="#">
+          <p>Amine Smahi</p>
+        </a>
+      </div>
+      <nav>
+        <div class="menu-btn"><i class="fas fa-bars"></i></div>
+        <ul>
+          <li class="closeBtn"><i class="fas fa-times"></i></li>
+          <li><a href="#intro">Portoflio</a></li>
+          <li><a href="#about-me">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
```

Comment:
- [ ]   The hamburger menu is not following the design template. You need to stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to make the hamburger menu closely similar to the [original design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471) 😸 .

### Current mobile menu

<div align="center" >
<img src="https://user-images.githubusercontent.com/80895497/147291240-41ab7fb7-c4af-4920-8e05-7b65bf39e64c.png" alt="img" />
</div>

 ### Original design

<div align="center">
<img src="https://user-images.githubusercontent.com/80895497/147281867-b9fd3f2f-881f-4451-8092-a26c6c53f37e.png" alt="img" />
</div>

Reply:
Ok

Reply:
I saw that you fix this issue, well done, now your design looks much better according to the Figma design. 😄